.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+## Proposed changes
+
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue. -->
+
+## Release Notes
+<!-- Include the Milestone Number and a small description of your change that will be added to the changelog -->
+
+Milestone:
+
+Changelog:
+
+## Types of changes
+
+What types of changes does your code introduce to Zowe Explorer?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Updates to Documentation or Tests (if none of the other choices apply)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
+- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
+- [ ] PR Description is included
+- [ ] `npm run vscode:prepublish` has been executed
+- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
+- [ ] I have added unit tests that prove my fix is effective, that my feature works and it has test coverage
+- [ ] I have added integration tests that prove my fix is effective or that my feature works
+- [ ] I have tested it manually and there are no regressions found
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] Any PR dependencies has been merged and published (if appropriate)
+
+## PR Reviewer Checklist
```

Comment:
As a PR reviewer, I would prefer to look at just 1 set of checklist. 
I would still go through what the PR author marked as completed but I believe we do not need to duplicate the checklist.
As you mentioned above, it is already a reminder of what PR reviewers will look for before merging.